arbitrum-docs/run-arbitrum-node/sequencer/high-availability-sequencer-docs.mdx

```diff
@@ -0,0 +1,479 @@
+          }
+      },
+      "execution": {
+        "forwarding-target": "",
+        "sequencer": {
+          "enable": true,
+          "max-tx-data-size": 85000,
+          "max-block-speed": "250ms"
+        },
+        "caching": {
+          "archive": true
+        }
+      }
+      "http": {
+          "addr": "0.0.0.0",
+          "port": "8547",
+          "vhosts": "*",
+          "api": ["arb","personal","eth","net","web3","txpool"],
+          "corsdomain": "*"
+      },
+      "ws": {
+          "addr": "0.0.0.0",
+          "port": "8548",
+          "origins": "*",
+          "api": ["arb","personal","eth","net","web3","txpool"]
+      },
+      "persistent": {
+          "chain": "/home/user/data/"
+      }
+    }
+kind: ConfigMap
+metadata:
+  name: sequencer-nitro
+```
+
+#### Command Line Arguments and Environment Variables:
+
+```yaml
+- args:
+    - --conf.env-prefix=NITRO
+    - --conf.file=/config/config.json
+    - --metrics
+    - --metrics-server.addr=0.0.0.0
+    - --node.seq-coordinator.signer.symmetric-fallback
+    - --node.seq-coordinator.signer.symmetric-sign
+    - --node.seq-coordinator.signer.symmetric.signing-key=<secure_key>
+  command:
+    - /usr/local/bin/nitro
+  env:
+    - name: POD_NAME
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: metadata.name
+    - name: NITRO_NODE_SEQ__COORDINATOR.MY__URL
+      value: ws://$(POD_NAME).svc:8548
+```
+
+#### Volume and ConfigMap Mapping:
+
+```yaml
+volumeMounts:
+  - mountPath: /home/user/data/
+    name: data
+  - mountPath: /config/
+    name: config
+  - mountPath: /home/user/seqkeystore/sequencer-wallet.key
+    name: l3-sequencer-keystore
+    subPath: sequencer-wallet.key
+volumes:
+  - configMap:
+      defaultMode: 420
+      name: sequencer-nitro
+    name: config
+  - name: l3-sequencer-keystore
+    secret:
+      defaultMode: 420
+      secretName: l3-sequencer-keystore
+volumeClaimTemplates:
+  - apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      name: data
+    spec:
+      accessModes:
+        - ReadWriteOnce
+      resources:
+        requests:
+          storage: 1Ti
+      storageClassName: gp3
+      volumeMode: Filesystem
+```
+
+### 6. Sequencer Coordinator Manager
+
+To manage active sequencer selection, use the built-in sequencer coordinator UI:
+
+- Follow detailed instructions at: [Running a Sequencer Coordinator Manager](https://docs.arbitrum.io/node-running/how-tos/running-a-sequencer-coordinator-manager)
+- Use this interface to switch between sequencer replicas when needed manually
+- Configure permissions and access controls appropriately
+
+### 7. Batchposter Setup
+
+The Batchposter component's primary function is posting transaction batches to the parent chain.
+
+You need to configure it to connect to the Sequencer coordinator Redis but don't set this node in the Sequencer priority list in SQM (Sequencer Coordinator Manager) to prevent it to be active sequencer:
```

Comment:
```suggestion
You need to configure it to connect to the sequencer coordinator Redis, but don't set this node in the Sequencer priority list in SQM (Sequencer Coordinator Manager) to prevent it from being an active sequencer:
```